### PR TITLE
[new release] alt-ergo (4 packages) (2.5.1)

### PR DESCRIPTION
**Pull request by Halbaroth**
The Alt-Ergo SMT prover

- Project page: <a href="https://alt-ergo.ocamlpro.com/">https://alt-ergo.ocamlpro.com/</a>
- Documentation: <a href="https://ocamlpro.github.io/alt-ergo">https://ocamlpro.github.io/alt-ergo</a>

This minor release fixes a critical soundness bug with the bitvector primitive `bvnot`. See https://github.com/OCamlPro/alt-ergo/pull/819.

### CODE REVIEW

**Comment by Halbaroth** [comment]
Our release 2.5.0 contains a critical bug that we found one day after the release... This release fixes the bug but we want to prevent users from installing 2.5.0. I put a flag `avoid-version` in each opam files of the release 2.5.0. Is it correct?

**Comment by mseri** [comment]
You can use avoid version to still allow forced installation and `available: false` otherwise. You need to add an extra constraint with avoid version though. I’ll add it and merge since it is critical and all green

File: packages/alt-ergo-lib/alt-ergo-lib.2.5.0/opam [hunk 1]
```diff
@@ -11,6 +11,7 @@ authors: ["Alt-Ergo developers <alt-ergo@ocamlpro.com>"]
 homepage: "https://alt-ergo.ocamlpro.com/"
 doc: "https://ocamlpro.github.io/alt-ergo"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+flags: [ avoid-version ]
```
Comment (mseri) [inline packages/alt-ergo-lib/alt-ergo-lib.2.5.0/opam:14]:
```suggestion
available: opam-version >= "2.1.0"
flags: [ avoid-version ]
```

File: packages/alt-ergo-parsers/alt-ergo-parsers.2.5.0/opam [hunk 2]
```diff
@@ -11,6 +11,7 @@ authors: ["Alt-Ergo developers <alt-ergo@ocamlpro.com>"]
 homepage: "https://alt-ergo.ocamlpro.com/"
 doc: "https://ocamlpro.github.io/alt-ergo"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+flags: [ avoid-version ]
```
Comment (mseri) [inline packages/alt-ergo-parsers/alt-ergo-parsers.2.5.0/opam:14]:
```suggestion
available: opam-version >= "2.1.0"
flags: [ avoid-version ]
```

File: packages/alt-ergo-plugin-ab-why3/alt-ergo-plugin-ab-why3.2.5.0/opam [hunk 3]
```diff
@@ -11,6 +11,7 @@ license: "LGPL-2.1-only"
 homepage: "https://alt-ergo.ocamlpro.com/"
 doc: "https://ocamlpro.github.io/alt-ergo"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+flags: [ avoid-version ]
```
Comment (mseri) [inline packages/alt-ergo-plugin-ab-why3/alt-ergo-plugin-ab-why3.2.5.0/opam:14]:
```suggestion
available: opam-version >= "2.1.0"
flags: [ avoid-version ]
```

File: packages/alt-ergo/alt-ergo.2.5.0/opam [hunk 4]
```diff
@@ -9,6 +9,7 @@ authors: ["Alt-Ergo developers <alt-ergo@ocamlpro.com>"]
 homepage: "https://alt-ergo.ocamlpro.com/"
 doc: "https://ocamlpro.github.io/alt-ergo"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"
+flags: [ avoid-version ]
```
Comment (mseri) [inline packages/alt-ergo/alt-ergo.2.5.0/opam:12]:
```suggestion
available: opam-version >= "2.1.0"
flags: [ avoid-version ]
```

**Pull request merged**